app/src/main/jni/desmume/src/MMU.cpp

```diff
@@ -49,22 +49,22 @@
 #define FLOATING_SQRT
 
 #ifdef HAVE_NEON
+#if defined(__arm__)
```

Comment:
Very easily avoidable by just removing -DHAVE_NEON from ARMv8, this is extra code that we don't need. Although, I don't see any drawbacks from having this enabled on ARM64.